pkgs/by-name/ca/calibre-web/package.nix

```diff
@@ -137,7 +138,7 @@ python.pkgs.buildPythonApplication rec {
   pythonImportsCheck = [ "calibreweb" ];
 
   passthru = {
-    tests.calibre-web = nixosTests.calibre-web;
+    tests = lib.optionalAttrs stdenv.hostPlatform.isLinux { inherit (nixosTests) calibre-web; };
```

Comment:
I don't think I've ever seen this anywhere else in the tree.

Can you explain why you want to do this?

Reply:
It's [somewhat common](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs+%22lib.optionalAttrs+stdenv.hostPlatform.isLinux%22+AND+%22nixosTests%22&type=code) amongst multiplatform packages with nixosTests as package tests. The idea is to prevent attempts to build this test on systems that don't support it when using something like `nix-build -A <pkg>.passthru.tests`

Ofborg does this for example, and it causes evaluation failures on Darwin -- as shown in [this](https://github.com/NixOS/nixpkgs/runs/27433552608) run from the [previous PR](https://github.com/NixOS/nixpkgs/pull/326583) to this package